README.md

```diff
@@ -78,6 +78,117 @@ createAction('ADD_TODO')('Use Redux');
 
 `metaCreator` is an optional function that creates metadata for the payload. It receives the same arguments as the payload creator, but its result becomes the meta field of the resulting action. If `metaCreator` is undefined or not a function, the meta field is omitted.
 
+### `createActions(?actionMap, ?arrayDefTypes, ?optionMap)`
+
+Create multiple actions and return an object mapping from the type to the actionCreator. This eliminates some of the repetitiveness when creating many actions.
+
+If `actionMap` is provided as first argument, an action will be created for each item in the map. The key will be used for the type. The value will be treated as follows:
+
+ - if value is a function then it will be assumed to be the payloadCreator function
+ - if value is an object then it will accept as its keys, `payload` and `meta`, both of which are functions that will serve as the payloadCreator and/or metaCreator. If payloadCreator is missing the default payloadCreator is used. If metaCreator is missing then it will not use one.
+ - otherwise create a default actionCreator for the key type
+
+If an array of string is provided as `arrayDefTypes` it will be used to create default actionCreators using the string for the action type.
+
+If an `optionMap` object is supplied as last argument then it will adjust the options createActions uses in operation:
+
+ - `prefix` - string, prefix all action types with this prefix, default ''
+
+#### Example: simple default actions
+
+```js
+import { createActions } from 'redux-actions';
+const a = createActions(['ONE', 'TWO', 'THREE']);
+
+// is equivalent to
+const a = {
+  ONE: createAction('ONE'),
+  TWO: createAction('TWO'),
+  THREE: createAction('THREE')
+;}
+```
+
+#### Example: simple default actions with prefix
+
+```js
+import { createActions } from 'redux-actions';
+const a = createActions(['ONE', 'TWO', 'THREE'], { prefix: 'ns/');
+
+// is equivalent to
+const a = {
+  ONE: createAction('ns/ONE'),
+  TWO: createAction('ns/TWO'),
+  THREE: createAction('ns/THREE')
```

Comment:
Thanks for the reply.

I think a limitation of ignoring the prefix in the returned object is that you can't use de-structuring syntax for two classes of name-spaced actions in the same file

```js
const { ACTION_ONE, ACTION_TWO, ACTION_THREE } = createActions([
  'ACTION_ONE', 
  'ACTION_TWO', 
  'ACTION_THREE'
], { prefix: 'foo/' });

// collision
const { ACTION_ONE, ACTION_TWO, ACTION_THREE } = createActions([
  'ACTION_ONE', 
  'ACTION_TWO', 
  'ACTION_THREE'
], { prefix: 'bar/' });
```

I also still find the following invocation a little odd

```js
ACTION_ONE({ actionData: 1 })
```

This looks a little strange to me as we're calling an action creator, but it **looks** like an action type string (the upper-case snake-case style is the convention in Redux).

***

I have to say that I'm not sure that the `prefix` option will be useful to everyone using this library (I know it would be useful to some). 

Have you encountered anything to suggest that it's an requested enough feature to be worth embedding into our APIs?